library/std/tests/windows_unix_socket.rs

```diff
@@ -0,0 +1,85 @@
+#![cfg(windows)]
+#![feature(windows_unix_domain_sockets)]
+// Maybe we should also add unix uds tests?
```

Comment:
I don't think this comment belongs here, but it does seem like a good idea. Feel free to file a separate PR.